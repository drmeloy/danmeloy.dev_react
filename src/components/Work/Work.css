.Work {
  padding-left: 130px;
  min-height: 101vh;

  p {
    font-family: 'Raleway', sans-serif;
    font-size: 1.2em;
    width: 85vw;
    padding: 1% 5%;
    line-height: 1.3;
    color: white;
  }

  @media (min-width: 10px) and (max-width: 720px){

    p {
      font-size: .8em;
    }
  }

  @media (max-width: 1025px){
    padding: 1%;
  }

  .Buttons {
    display: flex;
    justify-content: space-evenly;

    @media (min-width: 10px) and (max-width: 720px){
      flex-wrap: wrap;
      justify-content: center;
      font-size: .8em;
      row-gap: .5em;
      column-gap: .5em;
    }

    .Button {
      border-radius: 8px;
      display: flex;
      align-items: center;
      justify-content: center;
      font-family: 'Raleway', sans-serif;
      font-size: .8em;
      color: white;
      background: #03120347;
      border: 1px solid;
      padding: 1%;
      text-align: center;
      transition: .3s;

      @media (min-width: 10px) and (max-width: 720px){
        margin: 1%;
      }
    }

    .ActiveButton {
      border-radius: 8px;
      font-family: 'Raleway', sans-serif;
      font-size: .9em;
      font-weight: bold;
      color: white;
      background: #009688;
      border: 1px solid;
      padding: 1%;
      text-align: center;
      transition: .3s;
      box-shadow:
        0 13.2px 7.3px rgba(0, 0, 0, 0.125),
        0 25.2px 20.2px rgba(0, 0, 0, 0.18),
        0 39.3px 48.5px rgba(0, 0, 0, 0.235),
        0 74px 161px rgba(0, 0, 0, 0.36);
        cursor: default;

        @media (min-width: 10px) and (max-width: 720px){
          font-size: .8em;
          font-weight: normal;
          display: flex;
          align-items: center;
        }
    }
  }

  background: linear-gradient(135deg, rgba(101, 215, 91, 0.26) 0%, rgba(101, 215, 91, 0.26) 2%,rgba(87, 200, 111, 0.26) 2%, rgba(87, 200, 111, 0.26) 7%,rgba(72, 185, 131, 0.26) 7%, rgba(72, 185, 131, 0.26) 16%,rgba(58, 170, 151, 0.26) 16%, rgba(58, 170, 151, 0.26) 24%,rgba(43, 154, 170, 0.26) 24%, rgba(43, 154, 170, 0.26) 67%,rgba(29, 139, 190, 0.26) 67%, rgba(29, 139, 190, 0.26) 72%,rgba(14, 124, 210, 0.26) 72%, rgba(14, 124, 210, 0.26) 100%),linear-gradient(135deg, rgba(96, 227, 174, 0.26) 0%, rgba(96, 227, 174, 0.26) 32%,rgba(111, 197, 164, 0.26) 32%, rgba(111, 197, 164, 0.26) 55%,rgba(126, 168, 154, 0.26) 55%, rgba(126, 168, 154, 0.26) 69%,rgba(141, 138, 145, 0.26) 69%, rgba(141, 138, 145, 0.26) 79%,rgba(156, 108, 135, 0.26) 79%, rgba(156, 108, 135, 0.26) 82%,rgba(171, 79, 125, 0.26) 82%, rgba(171, 79, 125, 0.26) 84%,rgba(186, 49, 115, 0.26) 84%, rgba(186, 49, 115, 0.26) 100%),linear-gradient(90deg, rgb(60, 34, 11) 0%, rgb(60, 34, 11) 3%,rgb(60, 51, 15) 3%, rgb(60, 51, 15) 4%,rgb(60, 68, 19) 4%, rgb(60, 68, 19) 12%,rgb(60, 85, 24) 12%, rgb(60, 85, 24) 40%,rgb(60, 101, 28) 40%, rgb(60, 101, 28) 69%,rgb(60, 118, 32) 69%, rgb(60, 118, 32) 88%,rgb(60, 135, 36) 88%, rgb(60, 135, 36) 100%);
}

.Project {
  margin-top: 5%;
  display: flex;
  align-items: center;
  padding: 2% 0 2% 3%;

  .Screenshot {
    max-width: 40%;
    height: auto;

    @media (max-width: 1025px){
      max-width: 75%;
      margin-bottom: 5%;
    }
  }

  .Icons {
    display: grid;
    grid-template-columns: repeat(6, 1fr);
    justify-items: center;
    align-items: center;
    width: 90%;
    padding-top: 3%;

    p {
      width: fit-content;
      padding-right: 0;
    }

    figure {
      display: flex;
      flex-direction: column;
      justify-content: center;
      align-items: center;
      position: relative;
      margin-left: 5%;
      cursor: default;
    }

    figcaption {
      position: absolute;
      font-family: 'Raleway', sans-serif;
      color: white;
      font-size: .6em;
      opacity: 0;
      transition: .3s;
    }

    figure:hover > img, figure:hover > a {
      opacity: .5;
      transition: .3s;
    }

    figure:hover > figcaption {
      opacity: 1;
      transition: .3s;
    }

    .Link {
      cursor: pointer;
    }
  }

  div {
    width: 50%;
    padding-left: 2%;
    display: flex;
    flex-direction: column;
    justify-content: space-around;
  }

  p, span {
    font-family: 'Raleway', sans-serif;
    color: white;
    font-size: 1rem;
    width: 100%;
  }

  a > img, figure > img {
    height: 5vh;
  }

  a {
    color: #ffffff8a;
  }

  a:hover {
    color: #0000ffad;
  }

  h1 {
    font-family: 'Abril Fatface', cursive;
    font-size: 1.5em;
    color: white;
  }

  @media (min-width: 10px) and (max-width: 720px){
    div {
      width: 100%;
      margin-bottom: 5%;
    }

    h1 {
      font-size: 1em;
    }

    p {
      font-size: .7em;
    }
  }

  @media (min-width: 721px) and (max-width: 1025px) {
    p > img, a > img, span > img {
      max-width: 50px;
    }
  }

  @media (max-width: 1025px){
    flex-direction: column;
    align-items: center;
    padding: 0;

    div {
      width: 90%;
      margin-bottom: 5%;
    }

    h1, p {
      text-align: center;
    }
  }
}