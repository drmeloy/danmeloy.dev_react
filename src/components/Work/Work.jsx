import React from 'react';
import styles from './Work.css';

export default function Work(){
  return (
    <>
      <section className={styles.Work}>
        <span>Selected projects:</span>
        {/* KiT */}
        <div className={styles.Portfolio}>
          <img src='../../../public/assets/kit.png' />
          <div>
            <h1>KiT: Keeping In Touch</h1>
            <p>A progressive web app for setting, tracking, and meeting communication goals. User can input contacts, set communication goals, receive push notification reminders, and view communication logs. Contacts are automatically sorted and styled based on proximity to communcation deadline.</p><br/>
            <p>Built with:
              <img src='../../../public/assets/react.png' alt='react icon'/>
              <img src='../../../public/assets/redux.png' alt='redux icon'/>
              <img src='../../../public/assets/mongodb.png' alt='mongodb icon'/>
              <img src='../../../public/assets/node.png' alt='node icon'/>
              <img src='../../../public/assets/css.png' alt='css icon'/>
            </p><br/>
            <p>View on:
              <a href='https://friendlier-staging.netlify.com/login' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
              <a href='https://github.com/band-m' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a>
            </p>
          </div>
        </div>

        {/* Knowoncology */}
        <div className={styles.Portfolio}>
          <img src='../../../public/assets/know.png' />
          <div>
            <h1>Knowoncology</h1>
            <p>A client-driven backend migration for a naturopathic cancer research non-profit database <a href='https://www.knowoncology.org/' target='_blank' rel='noopener noreferrer'>(www.knowoncology.org)</a> built with MongoDB, resulting in 10x faster search speed and more reliable search results.</p><br/>
            <p>Built with:
              <img src='../../../public/assets/mongodb.png' alt='mongodb icon'/>
              <img src='../../../public/assets/node.png' alt='node icon'/>
              <img src='../../../public/assets/js.png' alt='javascript icon'/>
              <img src='../../../public/assets/css.png' alt='css icon'/>
            </p><br/>
            <p>View on:
              <a href='https://knowoncology.herokuapp.com' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
              <a href='https://github.com/kNOWOncology/knowoncology/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a>
            </p>
          </div>
        </div>

        {/* Treks n Trails */}
        <div className={styles.Portfolio}>
          <img src='../../../public/assets/treksntrails.png' />
          <div>
            <h1>Treks n&apos; Trails</h1>
            <p>Hiking and camping tool that munges data from five different APIs to generate a list of hiking trails, campgrounds, weather, and directions from a current or specified location. Hikes can be filtered by difficulty and length and a user can favorite a hike for later reference.</p><br/>
            <p>Built with:
              <img src='../../../public/assets/postgres.png' alt='postgres icon'/>
              <img src='../../../public/assets/node.png' alt='node icon'/>
              <img src='../../../public/assets/js.png' alt='javascript icon'/>
              <img src='../../../public/assets/css.png' alt='css icon'/>
            </p><br/>
            <p>View on:
              <a href='https://treksntrails.herokuapp.com' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
              <a href='https://github.com/Plant-Parenthood/treks-n-tents' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a>
            </p>
          </div>
        </div>

        {/* Artist Search */}
        <div className={styles.Portfolio}>
          <img src='http://placekitten.com/400/300' />
          <div>
            <h1>Artist Search</h1>
            <p>Test</p>
          </div>
        </div>

        {/* Address Book */}
        <div className={styles.Portfolio}>
          <img src='../../../public/assets/addressbook.png' />
          <div>
            <h1>Address Book</h1>
            <p>Full-stack CRUD application that allows a user to input, save, view, edit, and delete their contacts. User authentication ensures privacy of contact information.</p><br/>
            <p>Built with:
              <img src='../../../public/assets/mongodb.png' alt='mongodb icon'/>
              <img src='../../../public/assets/node.png' alt='node icon'/>
              <img src='../../../public/assets/js.png' alt='javascript icon'/>
              <img src='../../../public/assets/css.png' alt='css icon'/>
            </p><br/>
            <p>View on:
              <a href='https://alchemyaddressesbook.herokuapp.com' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
              <a href='https://github.com/drmeloy/lab-15-address-book' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a>
            </p>
          </div>
        </div>

        {/* DREAMy */}
        <div className={styles.Portfolio}>
          <img src='../../../public/assets/dreamy.png' />
          <div>
            <h1>DREAMy</h1>
            <p>Postman clone with trippy aesthetics. A user can hit any REST API endpoint with the GET, POST, PUT, PATCH, and DELETE methods, as well as add customized authentication headers. History is logged, viewable, and revistable.</p><br/>
            <p>Built with:
              <img src='../../../public/assets/react.png' alt='react icon'/>
              <img src='../../../public/assets/js.png' alt='javascript icon'/>
              <img src='../../../public/assets/css.png' alt='css icon'/>
            </p><br/>
            <p>View on:
              <a href='https://drmeloy.github.io/lab-28-RESTy/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
              <a href='https://github.com/drmeloy/lab-28-RESTy/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a>
            </p>
          </div>
        </div>

        {/* Github Viewer */}
        <div className={styles.Portfolio}>
          <img src='http://placekitten.com/400/300' />
          <div>
            <h1>Github Viewer</h1>
            <p>Test</p>
          </div>
        </div>

        {/* Quixx */}
        <div className={styles.Portfolio}>
          <img src='../../../public/assets/quixx.png' />
          <div>
            <h1>Qwixx</h1>
            <p>A mobile-responsive JavaScript implementation of the popular Qwixx dice game <a href='https://gamewright.com/product/Qwixx' target='_blank' rel='noopener roreferrer'>(https://gamewright.com/product/Qwixx)</a> that speeds gameplay and doesn&apos;t rely on paper scorecards. Developed with face-to-face multiplayer in mind with each player&apos;s device acting as their scorecard.</p><br/>
            <p>Built with: 
              <img src='../../../public/assets/js.png' alt='javascript icon'/>
              <img src='../../../public/assets/html.png' alt='html icon'/>
              <img src='../../../public/assets/css.png' alt='css icon'/>
            </p><br/>
            <p>View on:
              <a href='https://i-16.github.io/qwixx/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
              <a href='https://github.com/I-16/qwixx' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a>
            </p>
          </div>
        </div>
      </section>
    </>
  );
}
