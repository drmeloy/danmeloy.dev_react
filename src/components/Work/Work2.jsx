import React, { useState } from 'react';
import styles2 from './Work2.css';
import { Element } from 'react-scroll';

export default function Work2(){
  const [project, setProject] = useState('');

  return (
    <Element id='Work' name='Work'>
      <section className={styles2.Work}>
        <div className={styles2.Header}>
          <p>Selected projects:</p>
        </div>
        <div className={styles2.Buttons}>
          <span onMouseOver={() => setProject('kit')} onClick={() => setProject('kit')} className={styles2.Button}>KiT: Keeping In Touch</span>
          <span onMouseOver={() => setProject('know')} onClick={() => setProject('know')} className={styles2.Button}>KNOWOncology</span>
          <span onMouseOver={() => setProject('tnt')} onClick={() => setProject('tnt')} className={styles2.Button}>Treks n&apos; Trails</span>
          <span onMouseOver={() => setProject('band')} onClick={() => setProject('band')} className={styles2.Button}>Band Search</span>
          <span onMouseOver={() => setProject('address')} onClick={() => setProject('address')} className={styles2.Button}>Address Book</span>
          <span onMouseOver={() => setProject('dreamy')} onClick={() => setProject('dreamy')} className={styles2.Button}>DREAMy</span>
          <span onMouseOver={() => setProject('github')} onClick={() => setProject('github')} className={styles2.Button}>Github Viewer</span>
          <span onMouseOver={() => setProject('qwixx')} onClick={() => setProject('qwixx')} className={styles2.Button}>Qwixx</span>
        </div>
        { project === 'kit' ? 
          <div className={styles2.Project}>
            <img src='../../../public/assets/kit.png' />
            <div>
              <h1>KiT: Keeping In Touch</h1>
              <p>A progressive web app for setting, tracking, and meeting communication goals. User can input contacts, set communication goals, receive push notification reminders, and view communication logs. Contacts are automatically sorted and styled based on proximity to communcation deadline.</p>
              <p>Built with:
                <img src='../../../public/assets/react.png' alt='react icon'/>
                <img src='../../../public/assets/redux.png' alt='redux icon'/>
                <img src='../../../public/assets/mongodb.png' alt='mongodb icon'/>
                <img src='../../../public/assets/node.png' alt='node icon'/>
                <img src='../../../public/assets/css.png' alt='css icon'/>
                <span>View on:
                  <a href='https://friendlier-staging.netlify.com/login' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                  <a href='https://github.com/band-m' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></span>
              </p>
            </div>
          </div>

          : project === 'know' ?
            <div className={styles2.Project}>
              <img src='../../../public/assets/know.png' />
              <div>
                <h1>Knowoncology</h1>
                <p>A client-driven backend migration for a naturopathic cancer research non-profit database <a href='https://www.knowoncology.org/' target='_blank' rel='noopener noreferrer'>(www.knowoncology.org)</a> built with MongoDB, resulting in 10x faster search speed and more reliable search results.</p>
                <p>Built with:
                  <img src='../../../public/assets/mongodb.png' alt='mongodb icon'/>
                  <img src='../../../public/assets/node.png' alt='node icon'/>
                  <img src='../../../public/assets/js.png' alt='javascript icon'/>
                  <img src='../../../public/assets/css.png' alt='css icon'/>
                  <span>View on:
                    <a href='https://knowoncology.herokuapp.com' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                    <a href='https://github.com/kNOWOncology/knowoncology/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></span>
                </p>
              </div>
            </div>

            : project === 'tnt' ?
              <div className={styles2.Project}>
                <img src='../../../public/assets/treksntrails.png' />
                <div>
                  <h1>Treks n&apos; Trails</h1>
                  <p>Hiking and camping tool that munges data from five different APIs to generate a list of hiking trails, campgrounds, weather, and directions from a current or specified location. Hikes can be filtered by difficulty and length and a user can favorite a hike for later reference.</p>
                  <p>Built with:
                    <img src='../../../public/assets/postgres.png' alt='postgres icon'/>
                    <img src='../../../public/assets/node.png' alt='node icon'/>
                    <img src='../../../public/assets/js.png' alt='javascript icon'/>
                    <img src='../../../public/assets/css.png' alt='css icon'/>
                    <span>View on:
                      <a href='https://treksntrails.herokuapp.com' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                      <a href='https://github.com/Plant-Parenthood/treks-n-tents' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></span>
                  </p>
                </div>
              </div>

              : project === 'band' ?
                <div className={styles2.Project}>
                  <img src='../../../public/assets/bandsearch.png' />
                  <div>
                    <h1>Band Search</h1>
                    <p>Single page application to search musical artists, view discographies, examine albums, and get song lyrics</p>
                    <p>
              Built with:
                      <img src='../../../public/assets/react.png' alt='react icon'/>
                      <img src='../../../public/assets/js.png' alt='javascript icon'/>
                      <img src='../../../public/assets/css.png' alt='css icon'/>
                      <span>View on:
                        <a href='https://drmeloy.github.io/Artist-Search/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                        <a href='https://github.com/drmeloy/Artist-Search' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></span>
                    </p>
                  </div>
                </div>

                : project === 'address' ? 
                  <div className={styles2.Project}>
                    <img src='../../../public/assets/addressbook.png' />
                    <div>
                      <h1>Address Book</h1>
                      <p>Full-stack CRUD application that allows a user to input, save, view, edit, and delete their contacts. User authentication ensures privacy of contact information.</p>
                      <p>Built with:
                        <img src='../../../public/assets/mongodb.png' alt='mongodb icon'/>
                        <img src='../../../public/assets/node.png' alt='node icon'/>
                        <img src='../../../public/assets/js.png' alt='javascript icon'/>
                        <img src='../../../public/assets/css.png' alt='css icon'/>
                        <span>View on:
                          <a href='https://alchemyaddressesbook.herokuapp.com' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                          <a href='https://github.com/drmeloy/lab-15-address-book' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></span>
                      </p>
                    </div>
                  </div>

                  : project === 'dreamy' ?
                    <div className={styles2.Project}>
                      <img src='../../../public/assets/dreamy.png' />
                      <div>
                        <h1>DREAMy</h1>
                        <p>Postman clone with trippy aesthetics. A user can hit any REST API endpoint with the GET, POST, PUT, PATCH, and DELETE methods, as well as add customized authentication headers. History is logged, viewable, and revistable.</p>
                        <p>Built with:
                          <img src='../../../public/assets/react.png' alt='react icon'/>
                          <img src='../../../public/assets/js.png' alt='javascript icon'/>
                          <img src='../../../public/assets/css.png' alt='css icon'/>
                          <span>View on:
                            <a href='https://drmeloy.github.io/lab-28-RESTy/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                            <a href='https://github.com/drmeloy/lab-28-RESTy/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></span>
                        </p>
                      </div>
                    </div>

                    : project === 'github' ?
                      <div className={styles2.Project}>
                        <img src='../../../public/assets/githubviewer.png' />
                        <div>
                          <h1>Github Viewer</h1>
                          <p>Application used to view Github profiles and their repositories, pull requests, and issues in one convenient space. Hits the Github API and utilizes Redux for state management.</p>
                          <p>Built with:
                            <img src='../../../public/assets/react.png' alt='react icon'/>
                            <img src='../../../public/assets/redux.png' alt='redux icon'/>
                            <img src='../../../public/assets/js.png' alt='javascript icon'/>
                            <img src='../../../public/assets/css.png' alt='css icon'/>
                            <span>View on:
                              <a href='https://drmeloy.github.io/lab-40-github-viewer/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                              <a href='https://github.com/drmeloy/lab-40-github-viewer/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></span>
                          </p>
                        </div>
                      </div>

                      : project === 'qwixx' ?
                        <div className={styles2.Project}>
                          <img src='../../../public/assets/quixx.png' />
                          <div>
                            <h1>Qwixx</h1>
                            <p>A mobile-responsive JavaScript implementation of the popular Qwixx dice game <a href='https://gamewright.com/product/Qwixx' target='_blank' rel='noopener noreferrer'>(https://gamewright.com/product/Qwixx)</a> that speeds gameplay and doesn&apos;t rely on paper scorecards. Developed with face-to-face multiplayer in mind with each player&apos;s device acting as their scorecard.</p>
                            <span>Built with: 
                              <img src='../../../public/assets/js.png' alt='javascript icon'/>
                              <img src='../../../public/assets/html.png' alt='html icon'/>
                              <img src='../../../public/assets/css.png' alt='css icon'/>
                              <p>View on:
                                <a href='https://i-16.github.io/qwixx/' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/webpage.png' alt='webpage icon'/></a>
                                <a href='https://github.com/I-16/qwixx' target='_blank' rel='noopener noreferrer'><img src='../../../public/assets/github.png' alt='github icon'/></a></p>
                            </span>
                          </div>
                        </div>
                        : ''
        }
      </section>
    </Element>
  );
}
