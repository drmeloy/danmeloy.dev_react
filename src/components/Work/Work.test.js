import React from 'react';
import { shallow } from 'enzyme';
import Work from './Work';

describe('Work component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Work />);
    expect(wrapper).toMatchSnapshot();
  });
});
