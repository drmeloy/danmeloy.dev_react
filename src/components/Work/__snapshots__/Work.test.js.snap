// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work component matches snapshot 1`] = `
<Element
  id="Work"
  name="Work"
>
  <section
    className="Work"
  >
    <p>
      Selected projects:
    </p>
    <div
      className="Buttons"
    >
      <span
        className="ActiveButton"
        onMouseOver={[Function]}
      >
        KiT: Keeping In Touch
      </span>
      <span
        className="Button"
        onMouseOver={[Function]}
      >
        KNOWOncology
      </span>
      <span
        className="Button"
        onMouseOver={[Function]}
      >
        Treks n' Trails
      </span>
      <span
        className="Button"
        onMouseOver={[Function]}
      >
        Band Search
      </span>
      <span
        className="Button"
        onMouseOver={[Function]}
      >
        Address Book
      </span>
      <span
        className="Button"
        onMouseOver={[Function]}
      >
        DREAMy
      </span>
      <span
        className="Button"
        onMouseOver={[Function]}
      >
        Github Viewer
      </span>
      <span
        className="Button"
        onMouseOver={[Function]}
      >
        Qwixx
      </span>
    </div>
    <div
      className="Project"
    >
      <img
        className="Screenshot"
        src="../../../public/assets/kit.png"
      />
      <div>
        <h1>
          KiT: Keeping In Touch
        </h1>
        <p>
          A progressive web app for setting, tracking, and meeting communication goals. User can input contacts, set communication goals, receive push notification reminders, and view communication logs. Contacts are automatically sorted and styled based on proximity to communcation deadline.
        </p>
        <div
          className="Icons"
        >
          <p>
            Built with:
          </p>
          <figure>
            <img
              alt="react icon"
              src="../../../public/assets/react.png"
            />
            <figcaption>
              React
            </figcaption>
          </figure>
          <figure>
            <img
              alt="redux icon"
              src="../../../public/assets/redux.png"
            />
            <figcaption>
              Redux
            </figcaption>
          </figure>
          <figure>
            <img
              alt="mongodb icon"
              src="../../../public/assets/mongodb.png"
            />
            <figcaption>
              MongoDB
            </figcaption>
          </figure>
          <figure>
            <img
              alt="node icon"
              src="../../../public/assets/node.png"
            />
            <figcaption>
              NodeJS
            </figcaption>
          </figure>
          <figure>
            <img
              alt="css icon"
              src="../../../public/assets/css.png"
            />
            <figcaption>
              CSS
            </figcaption>
          </figure>
        </div>
        <div
          className="Icons"
        >
          <p>
            View on:
          </p>
          <a
            href="https://friendlier-staging.netlify.com/login"
            rel="noopener noreferrer"
            target="_blank"
          >
            <figure
              className="Link"
            >
              <img
                alt="webpage icon"
                src="../../../public/assets/webpage.png"
              />
              <figcaption>
                WWW
              </figcaption>
            </figure>
          </a>
          <a
            href="https://github.com/band-m"
            rel="noopener noreferrer"
            target="_blank"
          >
            <figure
              className="Link"
            >
              <img
                alt="github icon"
                src="../../../public/assets/github.png"
              />
              <figcaption>
                Github
              </figcaption>
            </figure>
          </a>
        </div>
      </div>
    </div>
  </section>
</Element>
`;
